README.md

```diff
@@ -9,12 +9,79 @@
 
 <img src="assets/spectrum-css_illustration_desktop.png">
 
+## Quick start
+
+### Node/yarn
+Install the components you want along with their dependencies. Here's an example using the button component:
+
+```shell
+yarn add -DW @spectrum-css/tokens @spectrum-css/typography @spectrum-css/page @spectrum-css/icon @spectrum-css/button
+```
+
+Spectrum CSS components utilize custom properties in order to change themes and scales. For these to apply, a couple of classes need to be added to the document’s `<html>` tag based on the [visual language](https://github.com/adobe/spectrum-css?tab=readme-ov-file#visual-language), [scale](https://github.com/adobe/spectrum-css?tab=readme-ov-file#scales), and [theme](https://github.com/adobe/spectrum-css?tab=readme-ov-file#themes-colorstops) you wish to use. For example, the following code snippet will display styling for the default Spectrum visual language using medium scale and light color theme:
+
+```html
+<html class="spectrum spectrum--medium spectrum--light">
+```
+
+Use the `index.css` files in your project to include component styles and all of the global context variable definitions (Spectrum and Express contexts, background theme/colorstop, platform scaling, etc.) for the component (if you don't need all of the global context definitions, peek at the docs for [including assets](https://github.com/adobe/spectrum-css?tab=readme-ov-file#including-assets)). Use this file by including the stylesheet (plus the stylesheets for the dependencies) in the `<head>` tag:
```

Comment:
```suggestion
Use the `index.css` files in your project to include component styles and all of the global context variable definitions (Spectrum and Express contexts, background theme/colorstop, platform scaling, etc.) for the component. If you don't need all of the global context definitions, peek at the docs for [including assets](https://github.com/adobe/spectrum-css?tab=readme-ov-file#including-assets)). Use this file by including the stylesheet and the stylesheets for the dependencies in the `<head>` tag:
```
A few edits to remove some parenthesis.

I'm thinking that the sentence "_global context variable definitions (Spectrum and Express contexts, background theme/colorstop, platform scaling, etc.)_" might be a little wordy and may need some adjustment to the terminology? Should we say something about global custom properties from the "Spectrum CSS tokens" package? Or reference "[design tokens](https://spectrum.adobe.com/page/design-tokens/)" for the design system?

Reply:
Good point, this is wordy. I'm trying not to repeat too much here in the quick start guide that folks can find in the README. I tried to simplify this a bit and link to relevant parts of the README. Let me know if the change addresses your concerns!